symfony/framework-bundle/5.1/config/services.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+use App\Kernel;
+use Symfony\Component\DependencyInjection\Loader\Configurator as di;
+
+// This file is the entry point to configure your own services.
+// Files in the packages/ subdirectory configure your dependencies.
+
+return function (di\ContainerConfigurator $container, Kernel $kernel): void {
```

Comment:
perhaps make it static as with the routes

Reply:
agreed, updated.